tests/test_git/test_git_init.py

```diff
@@ -0,0 +1,47 @@
+import sublime
+import os
+from .common import GSCreateRepoTestCase
+from GitSavvy.core import git_command
+
+
+def tidy_path(path):
+    return os.path.realpath(os.path.normcase(path))
+
+
+class TestInit(GSCreateRepoTestCase, git_command.GitCommand):
+
+    def test_01_window_path(self):
+        self.assertEqual(
+            tidy_path(self._temp_dir),
+            tidy_path(sublime.active_window().folders()[0]))
+        self.assertEqual(
+            tidy_path(self._temp_dir),
+            tidy_path(self.window.folders()[0]))
+
+    def test_02_add_file(self):
+        readme = os.path.join(self.repo_path, "README.md")
+        f = open(readme, "w")
+        f.write("README")
+        f.close()
+        self.git("add", "README.md")
+        yield 100
+        self.git("commit", "-m", "Init")
+
+    def test_03_is_master(self):
+        branch = self.get_current_branch_name()
+        self.assertEqual(branch, "master")
+
+    def test_04_untrack_file(self):
+        foo = os.path.join(self.repo_path, "foo")
+        with open(foo, "w") as f:
+            f.write("foo")
+
+        self.assertEqual(
+            [len(x) for x in self.sort_status_entries(self.get_status())],
+            [0, 0, 1, 0])
```

Comment:
I'm not sure I understand what this list of integers mean. Can we make some kind of CONST table to make this more readable?